src/docfx/restore/RestoreGit.cs

```diff
@@ -97,16 +97,24 @@ private static async Task<List<(string href, string head)>> AddWorkTrees(string
 
             Task FetchOrCloneRepo()
             {
+                var gitConfigs =
+                    from http in config.Http
+                    where url.StartsWith(http.Key)
+                    from header in http.Value.Headers
+                    select $"-c http.{http.Key}.extraheader=\"{header.Key}: {header.Value}\"";
```

Comment:
>{http.Key}. [](start = 37, length = 11)

Is this part needed? Seems git only supports `http.extraHeader` [https://github.com/git/git/blob/a42d7b6a5b5addcce01f483547ad96825c7a96e0/Documentation/config.txt#L1733](https://github.com/git/git/blob/a42d7b6a5b5addcce01f483547ad96825c7a96e0/Documentation/config.txt#L1733)

Reply:
https://github.com/git/git/blob/master/Documentation/config.txt#L2052